system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -262,3 +261,97 @@
     }
   }
 }
+
+.topsite-form {
+  $input-border-color: #424F59;
+  $error-background-color: #D63920;
+  $error-shadow-color: rgba(231, 111, 92, 0.75);
```

Comment:
Is it possible to use the newly defined photon colours in variables._scss for these 3 colours?